Database/Corrections/sodQuestFixes.lua

```diff
@@ -204,6 +204,12 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -415936,
             [questKeys.zoneOrSort] = sortKeys.MAGE,
         },
+        [78287] = {
+            [questKeys.objectivesText] = {"Take the Arms Shipment to Master Wood in Stormwind City."},
+        },
+        [78297] = {
+            [questKeys.objectivesText] = {"Return the Signed Procurement Contract to Grizzby in Ratchet."},
+        },
```

Comment:
```suggestion
```

Looks like a mistake to add these to the corrections while the base quest already contains them

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5336/commits/30b25b0b20d87108fbcfac0b1a2fcd2a868ac914